lib/common/util.go

```diff
@@ -15,10 +15,23 @@ import (
 
 	"github.com/btcsuite/btcutil/base58"
 	uuid "github.com/satori/go.uuid"
+	"github.com/stellar/go/keypair"
 )
 
+var (
+	TIMEFORMAT_ISO8601 string = "2006-01-02T15:04:05.000000000Z07:00"
```

Comment:
Why is the format an actual date ? That's so weird

Reply:
@Geod24 date time format in Golang is weird, I agree. :)